jupiterone/modifiers.go

```diff
@@ -194,3 +196,37 @@ func (jsonIgnoreDiff) PlanModifyMap(ctx context.Context, req planmodifier.MapReq
 
 	resp.PlanValue = req.StateValue
 }
+
+// useEmptyListForNullPlanModifier is a plan modifier that sets the plan value to an empty list
```

Comment:
This allows for minimize terraform plan diffs if the data comes back as an empty list from the api.

This allows the nil type to be converted to: [] so we can minimize diffs unless absolutely necessary.